docs/databags.md

```diff
@@ -118,6 +118,11 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `[app]['download']['version']` - Expected [version number][app.conf] (if any) used to determine if a new app should be downloaded.
 * `[app]['files']` - Hash of files to manage under the "default" or "local" directory.
 * `[app]['files'][filename]` - Contents of a particular file to manage. It can take 3 values, a hash of stanzas -> key-value pairs (then written with the splunk template), a string (written as is), or nil / false (deleted). If the hash or string is empty, the file is also deleted.
+* `[app]['files'][filename][stanza][attribute]` - A particular attribute in a configuration file. Can be either a discrete value or a hash (parts described below)
+* `[app]['files'][filename][stanza][attribute]['value']` - Hash that is translated to a proc to determine how to load a value for this attribute at runtime. Required when determining a value through a proc.
+* `[app]['files'][filename][stanza][attribute]['value']['proc']` - Method name from the CernerSplunk::ConfTemplate::Value module to invoke to create the proc. Other keys may be required or optional here (or may come from the general context by default), these map directly to the keyword arguments of the method for generating the proc
```

Comment:
I think this last sentence makes more sense to be on the line above. So:
```
* `[app]['files'][filename][stanza][attribute]['value']` - Hash that is translated to a proc to determine how to load a value for this attribute at runtime. Required when determining a value through a proc.  Certain keys may be required or optional here (or may come from the general context by default), these map directly to the keyword arguments of the method for generating the proc.
* `[app]['files'][filename][stanza][attribute]['value']['proc']` - Method name from the CernerSplunk::ConfTemplate::Value module to invoke to create the proc.
```

same for the transform doc below.